Tasks/VsTest/taskinputparser.ts

```diff
@@ -319,6 +337,42 @@ function getTiaConfiguration(): models.TiaConfiguration {
     return tiaConfiguration;
 }
 
+function getToolsInstallerConfiguration(): models.ToolsInstallerConfiguration {
+    const toolsInstallerConfiguration = {} as models.ToolsInstallerConfiguration;
+    
+    tl.debug("Path to VsTest from tools installer: " + tl.getVariable(constants.VsTestToolsInstaller.PathToVsTestToolVariable));
+    toolsInstallerConfiguration.vsTestPackageLocation = tl.getVariable(constants.VsTestToolsInstaller.PathToVsTestToolVariable);
+
+    // get path to vstest.console.exe
+    var matches = tl.findMatch(toolsInstallerConfiguration.vsTestPackageLocation, "**\\vstest.console.exe");
+    if (matches && matches.length !== 0) {
+        toolsInstallerConfiguration.vsTestConsolePathFromPackageLocation = matches[0];
+    } else {
+        utils.Helper.publishEventToCi(AreaCodes.TOOLSINSTALLERCACHENOTFOUND, tl.loc('toolsInstallerPathNotSet'), 1041, false);
+        throw new Error(tl.loc('toolsInstallerPathNotSet'));
+    }
+
+    // get path to Microsoft.IntelliTrace.ProfilerProxy.dll (amd64)
+    var amd64ProfilerProxy = tl.findMatch(toolsInstallerConfiguration.vsTestPackageLocation, "**\\amd64\\Microsoft.IntelliTrace.ProfilerProxy.dll");
```

Comment:
if we don't find in amd64 lets find it in x64 as well. can we do this change

Reply:
done, but take a  look at this code again